libs/langchain/langchain/vectorstores/milvus.py

```diff
@@ -138,11 +141,11 @@ def __init__(
         self.consistency_level = consistency_level
 
         # In order for a collection to be compatible, pk needs to be auto'id and int
-        self._primary_field = "pk"
-        # In order for compatiblility, the text field will need to be called "text"
-        self._text_field = "text"
+        self._primary_field = primary_field
+        # In order for compatibility, the text field will need to be called "text"
+        self._text_field = text_field
         # In order for compatibility, the vector field needs to be called "vector"
-        self._vector_field = "vector"
+        self._vector_field = vector_field
```

Comment:
Since these can now be set in in `__init__`, consider moving these comments to the `__init__` docstring as well? Seems like something users might want to know.

Reply:
I think we should do a pass over the code base with chat gpt at some point to re-write the doc-strings. it's pretty effective in tasks like these